### ISSUE TITLE
Bank object attributes don't match API JSON keys

### ISSUE THREAD

**Issue by javierjulio** [synapsepayments/synapse_pay-ruby#11]
I'm seeing several fields missing or keys not matching in name so several attribute values are null. All examples below performed in Sandbox mode.

An example JSON response when adding a bank account:

``` ruby
{
    : balance=>10000.0,
    : banks=>[
        {
            : account_class=>1,
            : account_number_string=>"***********4666",
            : account_type=>1,
            : address=>"",
            **: balance=>"",**
            : bank_name=>nil,
            : date=>"2015-05-20T17:22:57.095186",
            : email=>"",
            : id=>3186,
            : is_buyer_default=>true,
            : is_seller_default=>false,
            **: mfa_verifed=>false,**
            : name_on_account=>"Bobby Robson",
            : nickname=>"Example bank account",
            : phone_number=>"",
            : resource_uri=>nil,
            : routing_number_string=>"*******20"
        }
    ],
    : success=>true
}
```

And the object the library creates with missing attributes such as balance but also several null values for keys (is_active, is_verified) that don't match what's in the JSON above:

``` ruby
=> #<SynapsePay::Bank:0x3ffd6d3e94f0 id=3186> Attributes: {
  "account_class": 1,
  "account_number_string": "***********4666",
  "account_type": 1,
  "bank_name": null,
  "date": "2015-05-20T17:22:57.095186",
  "id": 3186,
  **"is_active": null,**
  "is_buyer_default": true,
  "is_seller_default": false,
  **"is_verified": null,**
  "name_on_account": "Bobby Robson",
  "nickname": "Example bank account",
  "resource_uri": null,
  "routing_number_string": "*******20"
}
```

**Comment by joncalhoun** [comment]
An update to the bank resource (the attributes in it specifically) seems to be the culprit here.

**Comment by joncalhoun** [comment]
This and all of the other issues you had should be resolved in the gem with a version of 0.0.3. Let me know if that doesn't work, but it should be good to go.
